pkg/tempo/config_test.go

```diff
@@ -0,0 +1,228 @@
+package tempo
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/config"
+	"gopkg.in/yaml.v3"
```

Comment:
Can this use `yaml.v2` instead so we don't have to vendor two different versions?